tests/models/glm4v_moe/test_modeling_glm4v_moe.py

```diff
@@ -294,11 +294,13 @@ def test_inputs_embeds_matches_input_ids(self):
 @require_torch
 @slow
 class Glm4vMoeIntegrationTest(unittest.TestCase):
-    model = None
+    @classmethod
+    def setUpClass(cls):
+        cls.model = None
 
     @classmethod
     def get_model(cls):
-        if cls.model is None:
+        if not hasattr(cls, "model") or cls.model is None:
```

Comment:
we don't need `not hasattr(cls, "model")`. Any reason you add this?

Reply:
Removed. CI Looks green.